src/fastfield/multivalued/reader.rs

```diff
@@ -36,15 +36,37 @@ impl<Item: FastValue> MultiValuedFastFieldReader<Item> {
         start..stop
     }
 
+    /// Returns `(start, stop)`, such that the values associated
+    /// to the given documents are `start..stop`.
+    #[inline]
+    fn range_multi(&self, doc: DocId, num: u32) -> Range<u64> {
+        let start = self.idx_reader.get(doc);
+        let stop = self.idx_reader.get(doc + num);
+        start..stop
+    }
+
+    /// Returns the array of values associated to the given `doc`s.
+    #[inline]
+    pub fn get_vals_multi(&self, doc: DocId, num: u32, vals: &mut Vec<Item>) {
```

Comment:
is that a range of docs?
Should we call it

```
fn get_vals_doc_range(&self, doc_range: Range<DocId>, vals: &mut Vec<Item>)
```

Reply:
I moved the code to the tests